index.html

```diff
@@ -309,7 +314,11 @@ <h2>Forms</h2>
                 <header>Listening to the Form</header>
                 <div class="col deer-view" deer-listening="formExample" deer-template="entity"></div>
             </div>
-
+<p>
+    There are many more complex ways to deal with forms to manage multiselects, arrays, and other types of 
```

Comment:
Hmm maybe clean this up a bit, I am not entirely sure what you are trying to say.  Maybe something like 

`There are many more complex ways forms acquire data through UI to accommodate more complex data objects that you may want to record.  How to process data from customized UIs is not well-documented yet, but we have many examples in our public code that we can direct you to upon request.`

Reply:
You write like a man who has slept. I accept this change.